src/main/java/org/opensearch/ad/rest/RestIndexAnomalyDetectorAction.java

```diff
@@ -53,26 +64,59 @@ public class RestIndexAnomalyDetectorAction extends AbstractAnomalyDetectorActio
 
     private static final String INDEX_ANOMALY_DETECTOR_ACTION = "index_anomaly_detector_action";
     private final Logger logger = LogManager.getLogger(RestIndexAnomalyDetectorAction.class);
-
-    public RestIndexAnomalyDetectorAction(Settings settings, ClusterService clusterService) {
-        super(settings, clusterService);
```

Comment:
Rather than changing the arguments here of super call, is there any way we can keep the arguments 
for settings -> environmentSettings
clusterService -> sdkClusterService

Reply:
Sure, we could use environment settings, but we have to "change the argument" for SDKClusterService anyway.  Both of those are available on the ExtensionsRunner, which we've somewhat started to standardize passing (along with the extension itself) as arguments to the rest handler constructors, to give us access to all the other things.

Once we have create components we may not need all the other things, though.

Definitely something to think about as I integrate create components.